test/nuts/mdapi.nut.ts

```diff
@@ -209,62 +216,107 @@ describe('mdapi NUTs', () => {
+          expect(reportCommandResponse).to.include('Status: Succeeded', reportCommandResponse);
+          expect(reportCommandResponse).to.include('Deployed: ', reportCommandResponse);
+        });
+
+        it('request verbose deploy report without a deployId', () => {
+          const reportCommandResponse = execCmd(
+            'force:mdapi:beta:deploy:report --wait 200 -u nonDefaultOrg --verbose',
+            {
+              ensureExitCode: 0,
+            }
+          ).shellOutput.stdout;
+          // has the basic table output
+          expect(reportCommandResponse).to.include('Deployed Source');
+        });
+      });
 
-  //       expect(reportCommandResponse).to.include('Status:  Succeeded', reportCommandResponse);
-  //       expect(reportCommandResponse).to.include('Components deployed:  2', reportCommandResponse);
-  //     });
-  //   });
+      // describe('Retrieve using non default username', () => {
+      //   it('should perform retrieve from the scratch org and request retrieve report', () => {
+      //     const retrieveCommandResponse = getString(
+      //       execCmd(
+      //         'force:mdapi:beta:retrieve --retrievetargetdir retrieveDir --unpackaged package.xml --wait 0 -u nonDefaultOrg',
+      //         { ensureExitCode: 0 }
+      //       ),
+      //       'shellOutput.stdout'
+      //     );
+      //     expect(retrieveCommandResponse).to.include(
+      //       'The retrieve request did not complete within the specified wait time'
+      //     );
 
-  //   describe('Retrieve using non default username', () => {
-  //     it('should perform retrieve from the scratch org and request retrieve report', () => {
-  //       const retrieveCommandResponse = getString(
-  //         execCmd(
-  //           'force:mdapi:retrieve --retrievetargetdir retrieveDir --unpackaged package.xml --wait 0 -u nonDefaultOrg',
-  //           { ensureExitCode: 0 }
-  //         ),
-  //         'shellOutput.stdout'
-  //       );
-  //       expect(retrieveCommandResponse).to.include(
-  //         'The retrieve request did not complete within the specified wait time'
-  //       );
+      //     const retrieveReportCommand = getString(
+      //       execCmd('force:mdapi:beta:retrieve:report --wait 2 -u nonDefaultOrg', {
+      //         ensureExitCode: 0,
+      //       }),
+      //       'shellOutput.stdout'
+      //     );
+      //     expect(retrieveReportCommand).to.include('Wrote retrieve zip to');
+      //   });
+      // });
 
-  //       const retrieveReportCommand = getString(
-  //         execCmd('force:mdapi:retrieve:report --wait 2 -u nonDefaultOrg', {
-  //           ensureExitCode: 0,
-  //         }),
-  //         'shellOutput.stdout'
-  //       );
-  //       expect(retrieveReportCommand).to.include('Wrote retrieve zip to');
-  //     });
-  //   });
+      describe('Deploy directory using non and request report using jobid parameter from wrong org', () => {
```

Comment:
This describe message is unclear.  "...using non...".  maybe, "Deploy directory to default org and request report from different org"?